src/kakarot/execution_context.cairo

```diff
@@ -32,7 +32,9 @@ namespace ExecutionContext {
         pedersen_ptr: HashBuiltin*,
         range_check_ptr,
         bitwise_ptr: BitwiseBuiltin*,
-    }(code: felt*, code_len: felt, calldata: felt*) -> model.ExecutionContext* {
+    }(
+        current_contract_address: Uint256*, code: felt*, code_len: felt, calldata: felt*
```

Comment:
Ethereum address is 20 bytes and starknet address is a felt so they both fit in a felt
```suggestion
        current_contract_sn_address: felt, code: felt*, code_len: felt, calldata: felt*
```